sealer/cmd/images.go

```diff
@@ -60,6 +60,7 @@ var listCmd = &cobra.Command{
 }
 
 func init() {
+	listCmd.DisableAutoGenTag = true
```

Comment:
We should apply the rule of DRY(Don't Repeat yourself). Actually doing command execution in every command's init() function is not a good way.

Reply:
Done. I see that only one command `rootCmd.DisableAutoGenTag=true` is enough. 

Dry is so important that avoid repetition of codes!